Documentation/BuildInstructionsMacOS.md

```diff
@@ -36,6 +36,16 @@ brew install llvm@18
 brew install gcc@13
 ```
 
+# Nix
+
+If Nix is installed, entering the devshell will provide an environment with all required build dependencies already included.
+
+```
+nix --extra-experimental-features "nix-command flakes" develop
+Meta/serenity.sh run
```

Comment:
At this point we usually just let the user return to the main build instructions document, so unless there is a need to we should omit the `serenity.sh` line.